docs/_docs/quickstart.md

```diff
@@ -4,25 +4,97 @@ title: Quick-start guide
 permalink: /docs/quickstart/
 ---
 
-For the impatient, here's how to get a boilerplate Jekyll site up and running.
+If you already have [Ruby](https://www.ruby-lang.org/en/downloads/) and [RubyGems](https://rubygems.org/pages/download) installed (see Jekyll's [requirements](/docs/installation/#requirements/)), you can create a new Jekyll site by doing the following:
 
 ```sh
-~ $ gem install jekyll bundler
-~ $ jekyll new myblog
+~ $ gem install jekyll bundler 
+# installs Jekyll and Bundler through RubyGems
+~ $ jekyll new myblog 
+# Creates a new Jekyll site called myblog
 ~ $ cd myblog
-~/myblog $ bundle exec jekyll serve
-# => Now browse to http://localhost:4000
+~/myblog $ bundle exec jekyll serve 
+# Builds the site on the preview server
+# Now browse to http://localhost:4000
 ```
 
-The `jekyll new` command now automatically initiates `bundle install` and installs the dependencies required. To skip this, pass `--skip-bundle` option like so `jekyll new myblog --skip-bundle`.
+The `jekyll new myblog` command installs a new Jekyll site into a subdirectory called `myblog`. The `jekyll new` command automatically initiates `bundle install` to install the dependencies required. (To skip `bundle install`, you can add the following flag: `jekyll new myblog --skip-bundle`.) {% comment %} why would someone want to skip bundle install -- seems unnecessary to even mention it. {% endcomment %}
 
```

Comment:
15-20 seconds when one has a blazing fast internet connection..
not everyone has a blazing fast connection.. (even if the connection is not as slow as yesteryear's dial-up speed..)
Fetching gem-index from rubygems can be slow partly due to the humongous no.of gems hosted there.. and  partly due to no.of dependencies for Jekyll itself.. (Your `Gemfile.lock` should give you a clue on the count..)